lib/locales/pt-BR.yml

```
@@ -692,6 +698,20 @@ pt-BR:
+
     gender:
       types: ["Feminino", "Masculino", "Não Binário", "Agênero", "Gênero Fluido", "Gênero queer", "Bigênero", "Poligênero"]
-      binary_types: ["Feminino", "Masculino"]
\ No newline at end of file
+      binary_types: ["Feminino", "Masculino"]
+
+    science:
+      element: ["Hidrogênio", "Hélio", "Lítio", "Berílio", "Boro", "Carbono", "Nitrogênio", "Oxigênio", "Flúor", "Néon", "Sódio", "Magnésio", "Alumínio", "Silício", "Fósforo", "Enxofre", "Cloro", "Argônio", "Potássio", "Cálcio", "Escândio", "Titânio", "Vanádio", "Cromo", "Manganês", "Ferro", "Cobalto", "Níquel", "Cobre", "Zinco", "Gálio", "Germânio", "Arsênico", "Selênio", "Bromo", "Criptônio", "Rubídio", "Estrôncio" ,"Ítrio", "Zircônio", "Nióbio", "Molibdênio", "Tecnécio", "Rutênio", "Ródio", "Paládio", "Prata", "Cádmio", "Índio", "Estanho", "Antimônio", "Telúrio", "Iodo", "Xenônio", "Césio", "Bário", "Lantânio", "Cério", "Praseodímio", "Neodímio", "Promécio", "Samário", "Európio", "Gadolínio", "Térbio", "Disprósio", "Hólmio", "Érbio", "Túlio", "Itérbio", "Lutécio", "Háfnio", "Tântalo", "Tungstênio", "Rênio", "Ósmio", "Irídio", "Platina", "Ouro", "Mercúrio", "Tálio", "Chumbo", "Bismuto", "Polônio", "Astatino", "Radônio", "Frâncio", "Rádio","Actínio", "Tório", "Protactínio", "Urânio", "Neptúnio", "Plutônio", "Amerício", "Cúrio", "Berquélio", "Califórnio", "Einstênio", "Férmio", "Mendelévio", "Nobélio", "Laurêncio", "Rutherfórdio", "Dúbnio", "Seabórgio", "Bóhrio", "Hássio", "Meitnério", "Darmstádtio", "Roentgênio", "Copernício", "Nipónio", "Fleróvio", "Moscóvio", "Livermório", "Tenesso", "Oganesson"]
+
+    space:
+      planet: ["Mercúrio", "Vênus", "Terra", "Marte", "Júpiter", "Saturno", "Urano", "Netuno"]
+      moon: ["Lua", "Fobos", "Deimos", "Europa", "Ganimedes", "Io", "Calisto", "Leda", "Himalia", "Lisiteia", "Mimas", "Tétis", "Encélado", "Dione", "Reia", "Jápeto", "Titã", "Cordélia", "Ofélia", "Miranda", "Umbriel", "Ariel", "Oberon", "Titânia", "Proteu", "Nereida", "Galateia", "Larissa", "Tritão"]
+      distance_measurement: ["anos-luz", "AU", "parsecs", "kiloparsecs", "megaparsecs"]
+
+    electrical_components:
+      active: ["Diodo", "LED", "Retificador", "Transistor", "JFET", "MOSFET", "Circuito Integrado", "LCD", "Tubo de raios catódicos", "Tubo de vácuo", "Bateria", "Célula de combustível", "Fonte de alimentação"]
+      passive: ["Resistor", "Potenciômetro", "TrimPot", "Varistor", "Capacitor", "Diodo Varicap", "Indutor", "Transformador", "Gerador", "Motor", "Solenóide", "Amplificador magnético", "Alto-falante", "Sensor de movimento", "Acelerômetro", "Termopar", "Antena", "Oscilador", "Placa de ensaio"]
+      electromechanical: ["Dispositivo piezoelétrico", "Cristal", "Motor ultrassônico", "Terminal", "Soquete", "Cabo de alimentação", "Chave de alternação", "Chave deslizante", "Chave DIP", "Pedal", "Teclado", "Circuit Breaker", "Fusível", "Placa de circuito impresso"]
```

🇺🇸 I believe the right translation of `Circuit Breaker` would be `Disjuntor`

🇧🇷 Acredito que a tradução de `Circuit breaker` seja `Disjuntor`

```suggestion
      electromechanical: ["Dispositivo piezoelétrico", "Cristal", "Motor ultrassônico", "Terminal", "Soquete", "Cabo de alimentação", "Chave de alternação", "Chave deslizante", "Chave DIP", "Pedal", "Teclado", "Disjuntor", "Fusível", "Placa de circuito impresso"]
```

---

Everything else looks great! Thanks

---

Eh sim 🇧🇷